components/test_raftstore/src/pd.rs

```diff
@@ -1665,8 +1665,8 @@ impl PdClient for TestPdClient {
                 )),
             )));
         }
-        let tso = self.tso.fetch_add(1, Ordering::SeqCst);
-        Box::pin(ok(TimeStamp::new(tso)))
+        let tso = self.tso.fetch_add(count as u64, Ordering::SeqCst);
+        Box::pin(ok(TimeStamp::new(tso + count as u64)))
```

Comment:
Why? For example, if count = 1 and self.tso is 1, it returns 1 in the past, but 2 now.

Reply:
In order to meet the conversion of PD that ["returns the timestamp with the biggest logical value"](https://github.com/tikv/tikv/blob/b5559844ce26a453900ca64c86902647dc807e28/components/pd_client/src/tso.rs#L210). And otherwise the [expected result](https://github.com/tikv/tikv/pull/12099/files#diff-e84ee9750d8a3e966a336e814bb4d3c2a9b88283afe12992c8045521551dd792R368) will be different with real PD.

I search codes may concern with the actual value of TSO, and change [one](https://github.com/tikv/tikv/pull/12099/files#diff-7875915f82a683a4d13b452ed136bcec8d572338c1e197b8a27142a40f6a618dR1066) of them.

Maybe I should add some comments for this new behavior of `set_tso`.